bin/run.py

```diff
@@ -119,7 +119,7 @@
             if v  not in decaylist: decaylist.append(v)
     
     sendjobGroup.add_argument('-d', '--decay', type=str, default='', help='pythia8 decay when needed', choices=decaylist)
-    sendjobGroup.add_argument('--pycard', type=str, default='p8_pp_default.cmd', help='pythia8 card')
+    sendjobGroup.add_argument('--pycard', type=str, default='p8_ee_default.cmd', help='pythia8 card')
```

Comment:
what changed here?

Reply:
There is no change in the card content. Just that `p8_pp` does not exist in the pythia directory for FCC-ee.

Reply:
ah, right, I didn't paid attention to ```ee``` wrt ```pp```